python/chronos/src/bigdl/chronos/forecaster/base_forecaster.py

```diff
@@ -83,15 +83,12 @@ def model_creator_orca(config):
                 optimizer = self.optimizer_creator(model, self.optim_config)
                 self.internal = Trainer.compile(model=model, loss=loss,
                                                 optimizer=optimizer)
-            self.onnxruntime_fp32 = None  # onnxruntime session for fp32 precision
-            self.openvino_fp32 = None  # placeholader openvino session for fp32 precision
-            self.jit_fp32 = None  # placeholader jit session for fp32 precision
-            self.onnxruntime_int8 = None  # onnxruntime session for int8 precision
-            self.openvino_int8 = None  # placeholader openvino session for int8 precision
-            self.jit_int8 = None  # placeholader jit session for int8 precision
-            self.pytorch_int8 = None  # pytorch model for int8 precision
-            self.optim_model = None  # accelarated model obtained from .optimize()
-            self.metadata = None  # str indicates model type of optim_model
+
+            self.accelerated_model = None  # accelerated model obtained from various accelerators
+            self.accelerate_method = None  # str indicates current accelerate method
+            self.quantize_models = {}      # quantization models, format: {model_type : model}
+            self.optim_model = None        # accelarated model obtained from .optimize() 
+            self.metadata = None           # str indicates model type of optim_model
```

Comment:
We could just merge `self.optim_model` to `self.accelerated_model ` and `self.metadata` to `self.accelerate_method `

Reply:
This means that `forecaster.predict(...)` will automatically use the accelerated model.

Reply:
> This means that `forecaster.predict(...)` will automatically use the accelerated model.

Does this mean `forecaster.predict()` will always use the latest accelerated model no matter what type it is ?